lib/src/runner/browser/browser.dart

```diff
@@ -108,7 +108,18 @@ abstract class Browser {
   /// exceptions.
   Future close() {
     _closed = true;
-    _process.then((process) => process.kill());
+
+    _process.then((process) {
+      // Dartium has a difficult time being killed on Linux. To ensure it is
+      // properly closed, find all children processes and kill those first.
+      if (Platform.isLinux) {
+        var result = Process.runSync('pgrep', ['-P', '${process.pid}']);
```

Comment:
I'd wrap this whole thing in `try...catch`. Possible that an exec doesn't exist on the target platform or security...who knows...